<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tampil</class>
 <widget class="QMainWindow" name="Tampil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="20" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>hirarki tertinggi</string>
      </property>
     </widget>
    </item>
    <item row="12" column="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>ReqBaru</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Sofhaliza</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
    </item>
    <item row="20" column="2">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="21" column="2">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>nama data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>hirarki tertinggi</string>
      </property>
     </widget>
    </item>
    <item row="12" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="17" column="3">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Server</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="20" column="3">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>ReqBaru</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>nama data</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QTableView" name="tableView_2"/>
    </item>
    <item row="15" column="0">
     <widget class="QTableView" name="tableView_5"/>
    </item>
    <item row="17" column="2">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="13" column="0" colspan="4">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="15" column="2" colspan="2">
     <widget class="QTableView" name="tableView_6"/>
    </item>
    <item row="21" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Ambil st untuk pengukuran baru</string>
      </property>
     </widget>
    </item>
    <item row="15" column="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Haliza</string>
      </property>
     </widget>
    </item>
    <item row="21" column="3">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>ReqBaru</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QTableView" name="tableView_3"/>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="TreeSubwin" name="treeView"/>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QTableView" name="tableView_4"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeSubwin</class>
   <extends>QTreeView</extends>
   <header>treesubwin.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
